test_runner/regress/test_storage_controller.py

```diff
@@ -4434,6 +4434,53 @@ def secondary_at_dest():
         assert initial_ps.http_client().tenant_list_locations()["tenant_shards"] == []
 
 
+def test_attached_0_graceful_migration(neon_env_builder: NeonEnvBuilder):
+    neon_env_builder.num_pageservers = 4
+    neon_env_builder.num_azs = 2
+
+    neon_env_builder.disable_kick_secondary_downloads = True
```

Comment:
nit: it's more difficult to understand inverted boolean options (like "disable")
And later this option converts to `kick_secondary_downloads`, which is even more frustrating
Let's call it the same `kick_secondary_downloads` everywhere